packages/babel-plugin-transform-es2015-template-literals/src/index.js

```diff
@@ -18,7 +18,9 @@ export default function ({ types: t }) {
         let raw = [];
 
         for (const elem of (quasi.quasis: Array)) {
-          strings.push(t.stringLiteral(elem.value.cooked));
+          if (elem.value.cooked !== null) {
```

Comment:
It should be present with value undefined. Length is still the same as that of .raw.